CHANGELOG.asciidoc

```diff
@@ -33,6 +33,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 
 *Affecting all Beats*
 
+- Keep a mapping of autodiscover instances and deployed configs. {pull}8851[8851]
```

Comment:
Maybe this could be a bit more clear? What effect will users notice? If none, we should mention that this is an internal refactor to add new features later.